srtcore/logging.h

```diff
@@ -166,14 +166,20 @@ struct SRT_API LogDispatcher
 
         // See Logger::Logger; we know this has normally 2 characters,
         // except !!FATAL!!, which has 9. Still less than 32.
```

Comment:
I think you have completely ignored what is written in this comment.

Here you can safely use `strncpy` with length 10 without worrying about the terminating 0.

Reply:
If I predicted this would happen, I'd better use `std::string` for this first part. This way you can also use `string::copy` for that part. I would have to review it a bit more to refax it.

Reply:
Indeed `std::string` would be better here, but let's go the way @oviano proposes here then.